extras/ams_wrapper/src/api/models/model.py

```diff
@@ -28,43 +28,34 @@
 from src.logger import get_logger
 from src.preprocessing.preprocess_image import preprocess_binary_image as default_preprocessing
 from src.api.ovms_connector import OvmsUnavailableError, ModelNotFoundError, OvmsConnector
-from src.api.models.input_config import ModelInputConfiguration, \
-    ModelInputConfigurationSchema, ValidationError
+from src.api.models.model_config import ModelInputConfiguration, \
+    ModelInputConfigurationSchema, ValidationError, ModelOutputConfiguration, \
+    ModelOutputConfigurationSchema
 
 logger = get_logger(__name__)
 
 
 class Model(ABC):
     def __init__(self, model_name: str, ovms_connector: OvmsConnector,
-                 labels_path: str, config_file_path: str = None):
+                 config_file_path: str = None):
         self.ovms_connector = ovms_connector
         self.model_name = model_name
-        self.labels = self.load_labels(labels_path)
-        self.input_configs: Dict[str, ModelInputConfiguration] = None # load input configuration
-        if config_file_path:
-            self.input_configs = self.load_input_configs(config_file_path)
-
-    def load_labels(self, labels_path):
-        try:                                                                          
-            with open(labels_path, 'r') as labels_file:
-                data = json.load(labels_file)
-                labels = dict()
-                for output in data['outputs']: 
-                    labels[output["output_name"]] = output['classes']
-        except Exception as e:                                                        
-            logger.exception("Error occurred while opening labels file: {}".format(e))
-            sys.exit(1)
-        return labels
+        #self.labels = self.load_labels(config_file_path)
+        self.input_configs = self.load_input_configs(config_file_path)
+        self.output_configs = self.load_output_configs(config_file_path) 
+        self.labels = {output_name: {index: label for label, index in self.output_configs[output_name].classes.items()}
+                       for output_name in self.output_configs.keys()}
```

Comment:
I'm not sure if we still need separate `labels` property - for now I've left it as it is used quite often in `Model` subclasses

Reply:
Lets leave it for clarity.